packages/hagrid/hagrid/cli.py

```diff
@@ -1244,6 +1255,11 @@ def create_launch_cmd(
     if parsed_kwargs["dev"] is True:
         parsed_kwargs["release"] = "development"
 
+    # choosing deployment type
+    parsed_kwargs["deploy"] = "container_stack"
```

Comment:
Same here, rename to `deployment_type` for consistency, as we use it in orchestra